lib/testing/src/lib/core/test-element.ts

```diff
@@ -98,6 +98,10 @@ export class TestElement {
         return BrowserActions.getText(this.elementFinder);
     }
 
+    async getInputValue(): Promise<string> {
```

Comment:
you don't need `async` here if you don't use `await` inside, returning Promise is absolutely fine
```suggestion
    getInputValue(): Promise<string> {
```

Reply:
async removed, as suggested